packages/components/src/Glimmer/style/Shape.css

```diff
@@ -0,0 +1,20 @@
+.rectangle {
+  width: 100%;
+}
+
+.rectangleShort {
+  width: 85%;
+}
+.rectangleShorter {
+  width: 70%;
+}
```

Comment:
Not really in love with how shorter glimmers can be defined tho so if you have another idea, let me hear it!